test/Microsoft.ML.Tests/Transformers/CategoricalTests.cs

```diff
@@ -297,12 +283,15 @@ public void TestCommandLine()
         [Fact]
         public void TestOldSavingAndLoading()
         {
-            var data = new[] { new TestClass() { A = 1, B = 2, C = 3, }, new TestClass() { A = 4, B = 5, C = 6 } };
+            var data = new[] {
+                new TestClass() { A = 1, B = new int[2] { 2, 3 }, C = new int[2] { 3, 4 } },
+                new TestClass() { A = 4, B = new int[2] { 2, 4 }, C = new int[3] { 2, 4, 3 } }
+            };
             var dataView = ML.Data.LoadFromEnumerable(data);
             var pipe = ML.Transforms.Categorical.OneHotEncoding(new[]{
-                    new InputOutputColumnPair("TermA", "A"),
-                    new InputOutputColumnPair("TermB", "B"),
-                    new InputOutputColumnPair("TermC", "C")
+                    new OneHotEncodingEstimator.ColumnOptions("CatA", "A"),
```

Comment:
`OneHotEncodingEstimator.ColumnOptions` is an internal class.  Can we instead write the tests using the public API?